### DISCUSSION TITLE
Scavenger Hunt Mission: Where in the world? 🌍

### DISCUSSION THREAD

**Discussion by elisemoe** [education/hackconviii#4] [General]
Major League Hacking is a global community of hackers and community builders, and we are so excited that this year's HackCon is accessible from anywhere. Share something about your part of the world by taking a picture that represents your culture or location.

**Comment by sampoder** [comment]
![20200816_204416](https://user-images.githubusercontent.com/39828164/90955785-02793c00-e4b3-11ea-8c97-82eadbf375f7.jpg)

I'm coming from Singapore!

**Comment by csleong98** [comment]
<img width="606" alt="Screen Shot 2020-08-22 at 8 13 02 PM" src="https://user-images.githubusercontent.com/6150058/90955875-eaee8300-e4b3-11ea-9c84-e7a6563d1c84.png">
Coming from Malaysia and eating is part our culture!!! Malaysia is a unique place where we have different races leaving together but we love the food from each other as well. The picture is a Malay dish (aka: Nasi Lemak)which is enjoyed by almost all Malaysian regardless of age, race or religion.

**Comment by charliebaer** [comment]
<img src="https://user-images.githubusercontent.com/50674268/90956722-b88a5900-e4a6-11ea-9830-bb58cdc6d503.png" width="600" height="800">

### Hyderabad - The land of the Authentic Biryani and the City of Nizams

**Comment by radhikasethi2011** [comment]
At home, celebrating ganesh chaturthi - Pune, India

![IMG_20200822_112615](https://user-images.githubusercontent.com/44310177/90956977-b4f7d180-e4a8-11ea-93e2-5eaa0118ad00.jpg)

**Comment by usamaeltmsah** [comment]
I'm from Egypt, the country of the great pyramid :egypt:  :100: 

![image](https://user-images.githubusercontent.com/23380057/90957039-b49d0d80-e48b-11ea-80f1-e08ba370f1ce.png)

**Comment by KhanhDoHoang** [comment]
![118309491_795386007933169_8842810739673370147_n](https://user-images.githubusercontent.com/55791090/90957104-c82f8080-e459-11ea-9ff5-a4de1721a8b2.jpg)

I am from Canada

**Comment by jaypatel15406** [comment]
Hey MLH Devs,
I am From India 🇮🇳  and India has Such a Diverse and Religious Culture. And According to Me, Temple was a key for Defining a Great and Rich Culture 🙏 
This was a Mahadev Temple Image Clicked By Me 👍 
![IMG_20190528_153223](https://user-images.githubusercontent.com/50138179/90959984-01004180-e4bc-11ea-95e0-34a1f99e4114.jpg)

**Comment by mohdsaabith** [comment]
i'm from kerala india !!
![IMG_1219](https://user-images.githubusercontent.com/44156514/90960089-d2cf3180-e4bc-11ea-9a59-aa60bd63128d.jpg)

**Comment by HrithikSahu** [comment]
Humans may have built towns and cities, but we aren’t the only ones who live in them. Given the smallest chance – a park, a garden, a window box, a basement, a subway tunnel, a bridge – wildlife manages to survive in the city. Look around – you may discover you’re in an urban jungle!

ps - i'm So proud of this picture also I'm From Mumbai, India

![photo-1547218321-b423e997ba77](https://user-images.githubusercontent.com/61985253/90962203-ff3e7a00-e4cb-11ea-9f0d-aa656ae2a1aa.jpg)

**Comment by annaarroyo** [comment]
i'm from austin, tx, usa and i love my city! although it's super hot right now, my family lives here and i am happy to be with them now during this pandemic!

![city-of-austin](https://user-images.githubusercontent.com/43049986/90963994-5c154280-e482-11ea-8005-95145f6eedfd.jpg)

**Comment by LilyPerr** [comment]
![hula](https://user-images.githubusercontent.com/42851016/90964127-caaace00-e48b-11ea-9e21-00eb115da006.jpg)
I'm from Hawai'i, so here's a picture of hula (from a simple Google search haha)

**Comment by BStarcheus** [comment]
![IMG_6766](https://user-images.githubusercontent.com/41553156/90964719-33487980-e491-11ea-9456-a4c1407012a2.JPG)
Midwest US. Lots of farmland. Usually overcast.

**Comment by dipakparmar** [comment]
From Beautiful British Columbia 🍁 
![](https://www.betterplaceimmigration.com/wp-content/uploads/2020/03/british-columbia.jpg)

**Comment by mike-taylor99** [comment]
East Coast US
![download (1)](https://user-images.githubusercontent.com/47544533/90966132-d94db100-e49c-11ea-8609-ffa152a0a1b0.jpg)

**Comment by Raymo111** [comment]
Canada!
![20171227_121642](https://user-images.githubusercontent.com/34818275/90966839-67c63080-e4a5-11ea-8e9b-ffd61f915d96.jpg)

**Comment by PaHorton** [comment]
The ~~scorching~~ beautiful state of Arizona! We've got lots of rocks here
![image](https://user-images.githubusercontent.com/13230281/90970536-44a67b80-e4bb-11ea-9ec2-ece299f46f66.png)

**Comment by arijitdas123student** [comment]
Hey folks. Happy to be here. Me from India. 🇮🇳🇮🇳

**Comment by BazilNizam** [comment]
![featured-image](https://user-images.githubusercontent.com/22983667/90976707-c00c3980-e55c-11ea-84e6-066cdf9a8f3f.jpg)
KERALA

**Comment by Garoto0518** [comment]
![antonio-lopez-Xx0_VT7Vx5I-unsplash](https://user-images.githubusercontent.com/22475754/90984797-69473580-e545-11ea-9ac1-48636fa6ee04.jpg)

Gainesville, Fl
Go Gators!

**Comment by 0xjba** [comment]
![photo6204168942012967634](https://user-images.githubusercontent.com/38902140/90985207-121e8080-e598-11ea-8509-33f1fff3b1ac.jpg)
@elisemoe There you go! Houseboats of Kerala. Do visit us.

**Comment by surajsahani** [comment]
![india-assam-14839497135o](https://user-images.githubusercontent.com/22853459/91024744-05d90880-e616-11ea-8570-c6b3a4c75ab3.jpg)
I am from India North-East(Assam)
Tea Garden of **_ASSAM_** and labours of Tea garden who can not do WFH.
